File: cs50x/week1/calculator/calculator.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask user for x 
    int x = get_int("What's x? ");

    // ask user for y
    int y = get_int("What's y? ");

    // print the result
    printf("%i\n", x + y);
    return 0;
}

File: CS50_Python/week0/calculator.py
class Calculator:
    def __init__(
        self,
    ) -> None:
        pass

    def add(self, x: float, y: float) -> float:
        return x + y

    def square(self, x: float) -> float:
        return x * x


def main() -> None:
    # get x and y from user
    x: float = float(input("What's X: ").strip())
    y: float = float(input("What's Y: ").strip())

    # calculate result
    cal: Calculator = Calculator()
    print(f"add result: {cal.add(x=x, y=y)}")
    print(f"Square of {x}: {cal.square(x=x)}")


if __name__ == "__main__":
    main()
